src/rf/apps/workers/process.py

```diff
@@ -122,7 +141,7 @@ def _validate_image(self, data):
 
             return updated
 
-    def _thumbnail(self, record):
```

Comment:
I don't think designating these steps as "internal use only" makes much sense in this case, because these are not implementation details of `QueueProcessor` that need to be hidden. It isn't completely unreasonable that we should be able to run these steps independently. Ex. `q = QueueProcessor(); q.thumbnail(...)`